build.gradle

```diff
@@ -19,7 +19,7 @@ allprojects {
 
     gradle.projectsEvaluated {
         tasks.withType(JavaCompile) {
-            options.compilerArgs << "-Xlint" << "-Xlint:-deprecation" << "-Xlint:-serial"
+            options.compilerArgs << "-Xlint"
```

Comment:
Can this be enabled by using android.lintOptions in build.gradle? Adding command line arguments manually does not look like the default way to go.